packages/tools/pie-icons/src/assets/qr-code.svg

```diff
@@ -0,0 +1,14 @@
+<svg width="17" height="16" viewBox="0 0 17 16" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
If it is a PIE icon it should have size 32x32 for large and 16x16 for small icons